src/Cli/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -793,4 +793,14 @@ The header is followed by the list of parameters and their errors (might be seve
     <value>Did you mean one of the following subcommands?</value>
     <comment>Followed by the list of suggestions</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="InstantiateCommand_Warning_FailedToGetTemplatePackageForTemplate" xml:space="preserve">
+    <value>Failed to get information about template packages for the template with identity '{0}'.</value>
+  </data>
+  <data name="InstantiateCommand_Warning_FailedToGetTemplatePackageForTemplateGroup" xml:space="preserve">
+    <value>Failed to get information about template packages for template group '{0}'.</value>
+  </data>
+  <data name="Commands_TemplateShortNameCommandConflict_Info" xml:space="preserve">
+    <value>To instantiate the template with short name '{0}', run:</value>
```

Comment:
Should there be any mention of fact that the command shortname is conflicting with other template and that's the reason why the shorthand expression cannot be used?

Reply:
@baronfel  what do you think?